components/recurring-contributions/UpdatePaymentMethodPopUp.js

```diff
@@ -94,7 +95,7 @@ const updatePaymentMethodMutation = gqlV2/* GraphQL */ `
 const addPaymentMethodMutation = gqlV2/* GraphQL */ `
   mutation addPaymentMethod($paymentMethod: PaymentMethodCreateInput!, $account: AccountReferenceInput!) {
     addStripeCreditCard(paymentMethod: $paymentMethod, account: $account) {
-      id
+      legacyId
```

Comment:
Is there a way to work with `id` instead if we're doing everything with GraphQL v2? What would be the blocker?

Reply:
Also as a general practice we should always pass an `id` (if available) when fetching data with Apollo. The main reason is that the id field is what allows Apollo to re-conciliate the cache, and also it prevents problems like opencollective/opencollective#1872

Reply:
This is being addressed in a separate PR